test suite fails

This is after a fresh install:

```bash
$ npm test
> jest && standard --fix && npm run lint && standard-markdown

 FAIL  ./test.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins

      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
          at Array.forEach (<anonymous>)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.372s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

@zeke do you have an idea what's going on?

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "jest": "^22.4.3",
     "semantic-release": "^15.1.7",
     "standard": "^11.0.1",
-    "standard-markdown": "^4.0.2",
+    "standard-markdown": "^5.0.1",
     "travis-deploy-once": "^4.4.1",
     "tslint": "^5.10.0",
     "typescript": "^2.8.3"
@@ -32,5 +32,8 @@
     "env": {
       "jest": true
     }
+  },
+  "jest": {
+    "testURL": "http://localhost"
   }
 }

